manage.py

```diff
@@ -1,57 +1,149 @@
 #!/usr/bin/env python
 
-import json
+
+import tabulate
 import typer
-from typing import List
+from typing import List, Optional
 from typing_extensions import Annotated
 from sqlmodel import Session
 from sqlmodel.sql.expression import select
 from fastapi import Depends
 from fastapi.params import Security
 from app.models.users import User, UserScopes
+from app.auth.security import get_password_hash
 from app.db import get_session
 from app.main import create_app
 
+
 app = typer.Typer()
 
-session: Session = next(get_session())
+
+def init_session(typer_app: typer.Typer, session: Session) -> None:
+    typer_app.db_session = session
 
 
 @app.command()
-def user_scopes_add(
+def add_scopes(
```

Comment:
Should this be add set_user_scopes so we don't get confused with scope diagnosis also it's replacing the whole scope?

Reply:
Good idea, I will change it to `add_user_scopes` to keep it consistent(add_x/list_x/update_x)